app/tailwind.config.js

```diff
@@ -188,7 +188,10 @@ module.exports = {
     divideColor: (theme) => theme('borderColor'),
     divideOpacity: (theme) => theme('borderOpacity'),
     divideWidth: (theme) => theme('borderWidth'),
-    fill: { current: 'currentColor' },
+    fill: (theme) => ({
+      current: 'currentColor',
+      primary: theme('colors.primary.500'),
```

Comment:
WHY? You can change all the svg fill by changing the `text-{color}-{number}` class